### PR TITLE
[#1267] Add PMD tests that contains unicode characters in source code

### PR DESCRIPTION
**Pull request by c71n93**
Related to #1267

### CODE REVIEW

File: qulice-pmd/src/test/java/com/qulice/pmd/PmdValidatorTest.java [hunk 1]
```diff
@@ -751,4 +751,17 @@ void allowsSwaggerAnnotations() throws Exception {
             )
         ).validate();
     }
+
+    /**
+     * PmdValidator can prohibit unicode characters in method names.
+     * @throws Exception If something wrong happens inside.
+     */
+    @Test
```
Comment (pnatashap) [inline qulice-pmd/src/test/java/com/qulice/pmd/PmdValidatorTest.java:759]: please add @DisabledOnOs(OS.Windows) for this test to be able to merge this PR

**Review by pnatashap — CHANGES_REQUESTED** [review]

**Comment by c71n93** [comment]
@pnatashap done

**Comment by pnatashap** [comment]
@yegor256 could you please merge this

**Comment by pnatashap** [comment]
@c71n93 a get your tests with the fix. please, close this PR
